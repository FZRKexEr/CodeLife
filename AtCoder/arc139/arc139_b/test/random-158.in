100
67 5 13 4 7 13
94 2 13 3 6 9
31 20 5 17 1 13
44 9 13 18 11 18
18 17 19 9 9 14
57 17 19 7 20 10
74 8 15 18 17 4
44 14 8 8 1 19
2 17 15 1 2 4
77 7 5 8 12 17
71 8 4 10 19 12
92 15 14 11 17 12
59 5 2 19 13 14
8 20 6 12 8 11
81 12 14 2 12 14
9 8 19 4 5 8
55 17 7 20 19 3
63 2 2 19 15 4
68 10 11 5 3 20
55 20 13 18 15 20
96 18 3 14 15 1
36 15 12 3 20 18
84 11 19 2 15 5
85 2 16 14 3 4
58 11 16 18 10 15
86 15 20 6 20 9
23 20 18 4 4 9
11 5 8 7 15 7
42 2 15 12 13 5
56 6 3 18 2 1
9 20 20 12 5 4
46 4 13 1 12 6
95 2 16 8 5 4
86 2 17 5 17 15
80 3 10 1 1 17
76 16 2 16 4 5
70 1 6 16 2 3
9 7 2 9 18 4
80 10 2 16 4 13
14 13 8 9 1 20
100 12 3 5 20 7
17 1 3 16 7 17
24 10 6 14 2 9
28 8 2 17 19 17
58 2 19 14 16 9
15 8 9 5 15 7
14 18 8 4 20 6
82 10 14 16 6 8
48 11 2 18 6 14
9 15 14 8 1 2
18 18 19 12 20 15
42 6 4 16 12 6
28 10 19 18 1 16
77 18 3 1 15 15
90 18 7 16 1 15
28 4 13 5 13 19
26 18 19 2 3 5
84 11 12 11 6 10
21 15 18 17 3 2
47 3 5 16 16 8
45 14 12 16 12 6
87 14 9 9 15 6
79 15 14 11 14 17
5 14 3 15 11 12
57 5 17 2 2 12
58 17 3 16 5 9
85 20 12 13 4 1
51 12 16 5 6 2
65 15 4 6 19 19
80 13 13 5 12 17
83 7 7 13 14 3
19 17 3 8 9 5
47 16 4 8 6 14
71 5 17 11 7 12
78 3 4 4 16 8
85 2 15 12 6 11
14 18 16 14 6 11
47 20 9 4 16 12
15 4 3 8 19 2
1 5 7 3 6 9
48 20 13 14 20 12
72 3 14 10 18 7
16 18 3 13 9 6
56 18 2 7 16 12
91 20 12 19 18 18
63 9 12 3 17 17
23 20 19 4 15 10
23 20 5 13 19 14
88 19 14 5 10 16
48 1 12 20 2 5
75 2 14 11 19 12
42 12 8 13 13 3
33 5 13 6 8 2
78 1 2 18 12 8
15 10 8 17 19 16
77 8 17 6 7 15
32 20 6 19 14 3
25 11 11 2 20 10
40 4 11 5 3 15
72 2 9 7 18 11
