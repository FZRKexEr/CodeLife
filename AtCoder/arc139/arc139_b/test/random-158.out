73
72
44
58
18
30
141
19
2
132
194
124
171
35
82
9
47
143
58
86
32
54
36
40
104
72
16
28
88
54
108
25
63
80
28
46
41
22
32
10
172
99
36
91
41
37
30
48
52
72
20
66
146
65
5
48
19
70
53
104
84
72
114
42
26
49
20
21
101
82
111
25
78
64
160
85
196
52
10
3
72
55
43
60
165
94
26
58
77
20
109
41
24
312
104
73
20
26
30
88
